### ISSUE TITLE
When building with the current alpine images, in Dec 2023, we get a "externally managed packages" issue from python

### ISSUE THREAD

**Issue by S4lt5** [Helioviewer-Project/helioviewer.org-docker#20]
Building on both linux and mac, the following occurs in CLI:

```
10.46 Executing busybox-1.36.1-r15.trigger
10.48 OK: 332 MiB in 71 packages
13.90 error: externally-managed-environment
13.90 
13.90 × This environment is externally managed
13.90 ??> 
13.90     The system-wide python installation should be maintained using the system
13.90     package manager (apk) only.
13.90     
13.90     If the package in question is not packaged already (and hence installable via
13.90     "apk add py3-somepackage"), please consider installing it inside a virtual
13.90     environment, e.g.:
13.90     
13.90     python3 -m venv /path/to/venv
13.90     . /path/to/venv/bin/activate
13.90     pip install mypackage
13.90     
13.90     To exit the virtual environment, run:
13.90     
13.90     deactivate
13.90     
13.90     The virtual environment is not deleted, and can be re-entered by re-sourcing
13.90     the activate file.
13.90     
13.90     To automatically manage virtual environments, consider using pipx (from the
13.90     pipx package).
13.90 
13.90 note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
13.90 hint: See PEP 668 for the detailed specification.
```

I added this before the PIP command, and everything worked great. 

This is covered @ https://stackoverflow.com/questions/75608323/how-do-i-solve-error-externally-managed-environment-every-time-i-use-pip-3

RUN rm /usr/lib/python3.11/EXTERNALLY-MANAGED

**Comment by dgarciabriseno** [comment]
Thanks for the report!

Looking at the discussion on stackoverflow, I think I'll try to resolve this by using a virtual environment.

**Comment by dgarciabriseno** [comment]
Fixed in d8317ec
